modules/ROOT/pages/2.4_Root_elements.adoc

```diff
@@ -127,6 +127,11 @@ In ESPD v3.0.0 the contracting party information will come from eForms. Which al
 
 *Rule*: (see examples further below in this document)
 
+|*cac:TenderingCriterion/cac:ProcurementProjectLotReference* |Associated class |0..n |One or more of the procurement project lots to which this criterion can be related to. a|
```

Comment:
this comment refers to commit above for the image. The picture in the documentation Figure 12 does not correspond to this diagram, for example 'TenderingCriterionXXXX**cbcID** in the documentaion but not in the diagram. 
I think the documentation needs update

Reply:
Diagram should be fixed now.